statistics/handle/ddl.go

```diff
@@ -39,10 +39,19 @@ func (h *Handle) HandleDDLEvent(t *util.Event) error {
 	case model.ActionAddColumn:
 		ids := getPhysicalIDs(t.TableInfo)
 		for _, id := range ids {
-			if err := h.insertColStats2KV(id, t.ColumnInfo); err != nil {
+			if err := h.insertColStats2KV(id, t.ColumnInfos[0]); err != nil {
 				return err
 			}
 		}
+	case model.ActionAddColumns:
+		ids := getPhysicalIDs(t.TableInfo)
+		for _, id := range ids {
+			for _, columnInfo := range t.ColumnInfos {
+				if err := h.insertColStats2KV(id, columnInfo); err != nil {
+					return err
+				}
+			}
```

Comment:
I will try this.

Reply:
Hi, I have move this iteration into `insertColStats2KV()`, but I have a quesion:

	case model.ActionAddColumns:
		ids := getPhysicalIDs(t.TableInfo)
		for _, id := range ids {
			if err := h.insertColStats2KV(id, t.ColumnInfos); err != nil {
				return err
			}
		}

`h.insertColStats2KV(id, t.ColumnInfos);` here maybe not good? Because for `model.ActionAddColumn` case is `h.insertColStats2KV(id, []*model.ColumnInfo{t.ColumnInfos[0]});`
@tangenta @zimulala

Reply:
Why not use the same code for `model.ActionAddColumn`? As follows:
```
case ActionAddColumn, model.ActionAddColumns:
		ids := getPhysicalIDs(t.TableInfo)
		for _, id := range ids {
			if err := h.insertColStats2KV(id, t.ColumnInfos); err != nil {
				return err
			}
		}
```